### ISSUE TITLE
This project is probably no longer needed

### ISSUE THREAD

**Issue by bd82** [brainkim/babel-plugin-chevrotain-serialize#4]
No more usage of Function.prototype.toString in Chevrotain 😄 
- https://sap.github.io/chevrotain/docs/changes/CHANGELOG.html#_6-0-0-8-20-2019

**Comment by brainkim** [comment]
This is fantastic news!

TODO on my end:
- [ ] deprecate npm package with info about how to upgrade
- [ ] archive this repo with info about how to upgrade

**Comment by bd82** [comment]
> This is fantastic news!

Yeah the Function.proto.toString caused too many issues.., and now with the new "grammar recording" style of analyzing it will be possible to implement previously blocked features 😄.
So it is worth a a few small breaking changes.

**Comment by brainkim** [comment]
Deprecated the npm package. I will wait a week or two before archiving the repository because 1. it’s permanent for some reason and 2. people may want to post comments in this issue about upgrading specific to this babel-transform.

**Comment by brainkim** [comment]
Archiving!
